tasks.py

```diff
@@ -534,6 +534,7 @@ def regenerate_package_mode(c, debug=False, swagger_group=None):
     azure_packages = [
         'test/azure/legacy/specification/packagemodemgmtplane/README.md',
         'test/azure/legacy/specification/packagemodecustomize/README.md',
+        'test/azure/legacy/specification/packagemodedataplane/README.md',
```

Comment:
I think there's already a packagemodedataplane, what's the difference between this one and the one in vanilla?

I think we can also do the package mode generations as part of the tasks.py file, instead of having a separate readme for each, can you look into that? Thanks!

Reply:
Hi @iscai-msft 
(1) template of README.md for dataplane will be rendered to result with different authentication: [azure-datalplane-has-aad-auth](https://github.com/Azure/autorest.python/blob/2735e03d3ce52b299acd52d9c89425a1d03c6dd2/test/azure/legacy/Expected/AcceptanceTests/PackageModeDataPlane/README.md), [vanilla-dataplane-no-aad-auth](https://github.com/Azure/autorest.python/blob/2735e03d3ce52b299acd52d9c89425a1d03c6dd2/test/vanilla/legacy/Expected/AcceptanceTests/PackageModeDataPlane/README.md). The added case is for better test coverage.
(2) our test case can promise better codegen quality but the structure of generated test SDK code is not very similar with product package so it is not suitable to set `package-mode` as regular test configuration(after try, I find we need to much work to update current test configuration like package-name, namespace, etc). The customized test case `regenerate-package-mode` for `package-mode` can better test different situation and the configuration is more similar with the real configuration(package-name, namespace, etc), so I think it is better to keep `regenerate-package-mode`